assembly/std/index.ts

```diff
@@ -140,28 +236,42 @@ export function generateEvent(event: string): void {
 /**
  * Transfers SCE coins from the current address to given address.
  *
- * @param to -
+ * @param to - the address to send coins to.
  * @param amount - value in the smallest unit.
+ *
+ * @throws
+ * - if the given address is not a valid address.
+ * - if the balance of the current address is insufficient to make the transaction.
+ *
  */
 export function transferCoins(to: Address, amount: u64): void {
   env.transferCoins(to.toString(), amount);
 }
 
 /**
- * Transfers SCE coins of the `from` address to the `to` address.
+ * Transfers SCE coins 'from' a given address 'to' another given address.
+ *
+ * @remarks
+ * The transfer is done only after approval.
```

Comment:
What do you mean by that ?

Reply:
Transfering coins from an other address than the current one might be protected ?
Didn't found any infos in massa-sc-runtime.

Reply:
@AurelienFT How do we know we have the right to use assembly_script_transfer_coins_for. For contract that we own ?

Reply:
You can only use this endpoint when you have the write access on the address you are passing as parameter. This is possible at SC address contract creation for example

Side note: SCE balance do no exist anymore they have been merged together since few months there is only one balance now.